testcases/commands/cron/cron_allow01

```diff
@@ -31,11 +31,16 @@ iam=`whoami`
 tvar=${MACHTYPE%-*}
 tvar=${tvar#*-}
 
-if [ "$tvar" = "redhat" -o "$tvar" = "redhat-linux" ]
-then
-CRON_ALLOW="/etc/cron.allow"
+# Support paths used by older distributions of RHEL or SLES
+if [ -f "/etc/cron.allow" ]; then
+	CRON_DENY="/etc/cron.deny"
+	CRON_ALLOW="/etc/cron.allow"
+elif [ -f "/var/spool/cron/allow" ]; then
+	CRON_DENY="/var/spool/cron/deny"
+	CRON_ALLOW="/var/spool/cron/allow"
 else
-CRON_ALLOW="/var/spool/cron/allow"
+	echo "Error while searching for cron access files"
+	exit 1
 fi
 
```

Comment:
I agree!

Would you like two new commits; the first to create the `cron_common.sh` and second to change the logic? Or can I simply make a new commit on top of this to refactor the code into the common file?